src/styles/styles.js

```diff
@@ -838,7 +835,7 @@ const styles = {
 
     textInputReversed: addOutlineWidth({
```

Comment:
Can you share a screenshot of where this one is used?

Reply:
Oh, good callout. I think this might not be used at all. Lemme check.

Reply:
Yeah these weren't used so I got rid of them.